lambdas/models/fhir/R4/nrl_fhir_document_reference.py

```diff
@@ -59,6 +62,7 @@ def build_fhir_dict(self):
                     }
                 }
             ],
+            "authenticator": {"identifier": {"value": self.custodian}},
             "content": [
                 {
                     "attachment": self.attachment.model_dump(
```

Comment:
reminder to self, these bits need bringing inline with changes made to the model in PRMP-1373 regarding mandatory fields etc